test/cpp/api/modules.cpp

```diff
@@ -326,6 +327,18 @@ TEST_F(ModulesTest, Linear2_CUDA) {
   ASSERT_EQ(model->weight.grad().numel(), 2 * 5);
 }
 
+TEST_F(ModulesTest, L1Loss) {
+  L1Loss loss;
+  auto input = torch::randn({5,6}, torch::requires_grad());
+  auto target = torch::empty({5,6}).random_(2);
+  auto output = loss->forward(torch::sigmoid(input), target);
+  auto s = output.sum();
+  s.backward();
+
+  ASSERT_EQ(output.sizes(), torch::IntArrayRef());
+  ASSERT_EQ(input.sizes(), input.grad().sizes());
```

Comment:
Is there a way we can also check the correctness of `output` value after `loss->forward`?

Reply:
I based this on python's lL1Loss test and I think that it didn't check. What is the expected value here?

Reply:
Got it, I think we can rely on the parity test to make sure the output value is in sync with the Python API.